scripts/build.ps1

```diff
@@ -249,7 +251,11 @@ if ($BuildDocumentation) {
 # Set default log dir if not specified.
 $runtime_dir = "$BuildOutputPath/bin"
 $library_dir = "$BuildOutputPath/lib"
+
 $cmake_bin = 'cmake.exe'
+$cmake_cmd = Get-Command cmake.exe
+$cmake_bin = $cmake_cmd.Path
+Write-Verbose ('Using $cmake_cmd.Path')
```

Comment:
parens not needed.

argument wont be expanded as you're not using ".
did you mean write-verbose ('using {0}' -f $make_cmd.Path) ?

Reply:
will address in next PR